pretty_printers_tests/src/main.rs

```diff
@@ -14,10 +16,32 @@ const SETTINGS: &str = "Settings_macos.toml";
 #[cfg(all(not(target_os = "linux"), not(target_os = "macos")))]
 panic!("Unsupported platform");
 
+#[derive(Deserialize)]
+struct Settings {
+    test_dir: String,
+    pretty_printers_path: String,
+    lldb_batchmode: String,
+    lldb_lookup: String,
+    python: String,
+    print_lldb_stdout: bool,
+    lldb_native_rust: bool,
+}
 
+/// Expects `lldb_python` path as the first argument
 fn main() -> Result<(), ()> {
     let settings = fs::read_to_string(SETTINGS).expect(&format!("Cannot read {}", SETTINGS));
-    let lldb_config: LLDBConfig = toml::from_str(&settings).expect(&format!("Invalid {}", SETTINGS));
+    let settings: Settings = toml::from_str(&settings).expect(&format!("Invalid {}", SETTINGS));
+    let lldb_python = get_lldb_python_path();
```

Comment:
Maybe place it at the beginning of the `main`? IMO there it's better to check argument before reading a file.

Also, it can be inlined like this (but it is a matter of taste):
```Rust
fn main() -> Result<(), ()> {
    let args: Vec<String> = std::env::args().collect();
    let lldb_python = args.get(1).expect("You need to pass a path to `lldb_python`").clone();
    let settings = fs::read_to_string(SETTINGS).expect(&format!("Cannot read {}", SETTINGS));
```